_posts/en/newsletters/2023-12-20-newsletter.md

```diff
@@ -0,0 +1,992 @@
+transfers, and paying LN invoices.  Concerns were raised about the high
+onchain footprint and the need for the operator to keep a large amount
+of funds in a hot wallet compared to LN.  However, several
+developers remained enthusiastic about the proposed protocol and its
+potential to provide a simple and trustless experience for its users.
+
+## June
+
+{:#silentpayments}
+Josie Baker and Ruben Somsen [posted][jun sp] a draft BIP for [silent
+payments][topic silent payments], a type of reusable payment code that
+will produce a unique onchain address each time it is used, preventing
+[output linking][topic output linking].  Output linking can
+significantly reduce the privacy of users (including users not directly
+involved in a transaction). The draft goes into detail about the
+benefits of the proposal, its tradeoffs, and how software can
+effectively use it.  Ongoing work implementing silent payments for
+Bitcoin Core was also [discussed][aug sp] during a Bitcoin Core PR
+Review Club meeeting.
+
+<div markdown="1" class="callout" id="security">
+### 2023 summary<br>Security disclosures
+
+Optech reported on three significant security vulnerabilities this year:
+
+- [Milk Sad vulnerability in Libbitcoin `bx`][aug milksad]: a widely
+  undocumented lack of entropy in a command suggested for creating
+  wallets ultimately led the theft of a significant number of
+  bitcoins across multiple wallets.
+
+- [Fake funding denial-of-service against LN nodes][aug fundingdos]: a
+  denial-of-service attack was privately discovered and [responsibly
+  disclosed][topic responsible disclosures] reported by Matt Morehouse.
+  All affected nodes were able to update and, as of this writing, we are
+  unaware of the vulnerability being exploited.
+
+- [Replacement cycling against HTLCs][oct cycling]: a fund-stealing
+  attack against [HTLCs][topic htlc] used in LN and possibly other
+  protocols was privately discovered and responsibly disclosed by
+  Antoine Riard.  All LN implementations tracked by Optech deployed
+  mitigations, although the effectiveness of those mitigations was a
+  subject of discussion and other mitigations have been proposed.
+
+</div>
+
+## July
+
+{:#vls}
+The Validating Lightning Signer (VLS) project [released][jul vls] their
+first beta version in July.  The project allows the separation of an LN
+node from the keys that control its funds. An LN node running with VLS
+will route signing requests to a remote signing device instead of local
+keys. The beta release supports CLN and LDK, layer-1 and layer-2
+validation rules, backup and recovery capabilities, and provides a
+reference implementation.
+
+{:#ln-meeting}
+An LN developer [meeting][jul summit] held in July discussed a variety
+of topics, including reliable transaction confirmation at the base
+layer, [taproot][topic taproot] and [MuSig2][topic musig] channels,
+updated channel announcements, [PTLCs][topic ptlc] and [redundant
```

Comment:
Could reference another (September) PTLC topic here, but a bit tortured perhaps:

- LN messaging changes for PTLCs - https://bitcoinops.org/en/newsletters/2023/09/13/#ln-messaging-changes-for-ptlcs

Reply:
Yeah, I thought about that but I got the impression (perhaps mistakenly) that people didn't find that discussion useful, so I decided not to draw attention to it.  (Insert here the meme "looks-at-you-looks-away")

Reply:
I think its fine to leave out